src/features/Students/StudentsMetrics/index.jsx

```diff
@@ -2,28 +2,34 @@ import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { Card, CardGrid, ToggleButton } from '@edx/paragon';
 import { ToggleButtonGroup } from 'react-paragon-topaz';
-import { fetchMetricsData } from 'features/Students/data/thunks';
+import { fetchClassesMetricsData, fetchStudentsMetricsData } from 'features/Students/data/thunks';
 
 import 'features/Students/StudentsMetrics/index.scss';
 
 const StudentsMetrics = () => {
   const dispatch = useDispatch();
-  const stateMetrics = useSelector((state) => state.students.metrics.data);
+  const institution = useSelector((state) => state.main.selectedInstitution);
+  const studentsMetrics = useSelector((state) => state.students.studentsMetrics.data);
+  const classesMetrics = useSelector((state) => state.students.classesMetrics.data);
+  const days = 7; // Temporal variable
 
   useEffect(() => {
-    dispatch(fetchMetricsData());
-  }, [dispatch]);
+    if (Object.keys(institution).length > 0) {
+      dispatch(fetchStudentsMetricsData(institution.id, days));
+      dispatch(fetchClassesMetricsData(institution.id, days));
+    }
+  }, [institution, dispatch]);
 
   return (
     <div className="container-cards d-flex flex-column">
       <ToggleButtonGroup type="radio" defaultValue={1} name="period">
         <ToggleButton id="tbg-radio-1" value={1} variant="outline-primary">
           This week
         </ToggleButton>
-        <ToggleButton id="tbg-radio-2" value={2} variant="outline-primary">
+        <ToggleButton id="tbg-radio-2" value={2} variant="outline-primary" disabled>
           Next week
         </ToggleButton>
-        <ToggleButton id="tbg-radio-3" value={3} variant="outline-primary">
+        <ToggleButton id="tbg-radio-3" value={3} variant="outline-primary" disabled>
```

Comment:
Same comment. @sergivalero20

Reply:
@anfbermudezme It's not possible add inline comments there.

Reply:
@sergivalero20 , you could try [this](https://github.com/Pearson-Advance/frontend-app-institution-portal/pull/45#discussion_r1515008667)